README.adoc

```diff
@@ -52,9 +52,13 @@ A FaaS for Kubernetes
 
 
 
-== Installation
+== Installation of latest release
 
-You can install the system using link:Getting-Started.adoc#helm[a Helm chart] or by link:#manual[manually building and deploying] the components.
+You can install the latest release using link:Getting-Started.adoc[a Helm chart]. 
+
+== Developer installation
+
+If you want to develop riff you can install the current development version with the link:Developing-with-Minikube.adoc[Helm chart] or by link:#manual[manually building and deploying] the components.
```

Comment:
Regarding my above comment, this is more to the point.
I guess then, that the only thing I'm not at ease with is the choice of words "Developing riff with minikube". Two giveaways that there is something phony IMO:
- You use "Helm Chart" as the link title
- The linked document isn't that specific about minikube, when you think about it

Reply:
Fixing